js/ace.js

```diff
@@ -0,0 +1,1046 @@
+        //
+        //     {
+        //         "attachment": 200,
+        //         "message": null,
+        //         "parameters": null,
+        //         "status": 200
+        //     }
+        //
+        return response;
+    }
+
+    async fetchOrder (id, symbol = undefined, params = {}) {
+        /**
+         * @method
+         * @name ace#fetchOrder
+         * @description fetches information on an order made by the user
+         * @see https://github.com/ace-exchange/ace-official-api-docs/blob/master/api_v2.md#open-api---order-status
+         * @param {string} symbol unified symbol of the market the order was made in
+         * @param {object} params extra parameters specific to the ace api endpoint
+         * @returns {object} An [order structure]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure}
+         */
+        await this.loadMarkets ();
+        const request = {
+            'orderNo': id,
+        };
+        const response = await this.privatePostV2OrderShowOrderStatus (this.extend (request, params));
+        //
+        //     {
+        //         "attachment": {
+        //             "buyOrSell": 1,
+        //             "averagePrice": "490849.75000000",
+        //             "num": "0.00000000",
+        //             "orderTime": "2022-11-29 18:03:06.318",
+        //             "price": "490849.75000000",
+        //             "status": 4,
+        //             "tradeNum": "0.02697000",
+        //             "remainNum": "0.97303000",
+        //             "baseCurrencyId": 2,
+        //             "baseCurrencyName": "BTC",
+        //             "quoteCurrencyId": 1,
+        //             "quoteCurrencyName": "TWD",
+        //             "orderNo": "16697161898600391472461100244406"
+        //         },
+        //         "message": null,
+        //         "parameters": null,
+        //         "status": 200
+        //     }
+        //
+        const data = this.safeValue (response, 'attachment');
+        return this.parseOrder (data, undefined);
+    }
+
+    async fetchOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        /**
+         * @method
+         * @name ace#fetchOrders
+         * @description fetches information on multiple orders made by the user
+         * @see https://github.com/ace-exchange/ace-official-api-docs/blob/master/api_v2.md#open-api---order-list
+         * @param {string} symbol unified market symbol of the market orders were made in
+         * @param {int|undefined} since the earliest time in ms to fetch orders for
+         * @param {int|undefined} limit the maximum number of  orde structures to retrieve
+         * @param {object} params extra parameters specific to the ace api endpoint
+         * @returns {[object]} a list of [order structures]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure}
+         */
+        if (symbol === undefined) {
```

Comment:
@sc0Vu does this method only return open orders? If so we need to rename it to `fetchOpenOrders`

```
Node.js: v17.9.0
CCXT v2.6.47
ace.fetchOrders (LTC/USDT)
2023-01-24T09:48:43.293Z iteration 0 passed in 437 ms

[]
0 objects
2023-01-24T09:48:43.293Z iteration 1 passed in 437 ms

```